WK2-1923 Add valuations guide (#106)
wealthkernel/api-documentation-guides: docs/valuations/Basics.md

---
tags: [Valuations]
---

# Valuations

A valuation is a breakdown of the cash and holdings in a portfolio, taking into account prices and FX rates. 

Although valuations are dated daily, transactions can be booked that affect existing valuations, causing their values to change. For example, an incorrectly dated Dividend transaction being rebooked for the correct date.

In other words, transactions with a timestamp on or before a valuation’s date will be reflected in the valuation, even if they are booked retrospectively days after the valuation date. This effect perpetuates through all valuations up to the current valuation.

## Concepts

> - New valuations are calculated for the previous day, i.e. are T+1. 

Reviewer: ```suggestion
- Valuations are made available on a T+1 basis.
```

- Valuations are designed to give a point in time snapshot of a portfolio. For more realtime data, [Balances](../balances/Basics.md) should be used.
- The `changedAt` property on a valuation shows the time that the most recent update to that valuation occurred.

## Examples

> On 2025-01-01 a deposit is made to prt-1. This deposit will first be reflected in valuation prt-1-20250101. This valuation will be calculated early in the morning (UTC) on 2025-01-02.

Author: UTC isn't exactly right, the cron is in London/Europe zone, so not sure what the best way to describe this is. Could be omitted, but I wanted to give some indication of what morning meant.

Reviewer 2: Maybe worth calling out that valuation creation could be delayed if the underlying data (i.e. prices and fx rates) have not yet arrived, or something to that effect.

Reviewer: This could be part of an FAQ e.g. Why hasn't my portfolios valuation been created yet?
